Cabal/doc/nix-local-build.rst

```diff
@@ -261,8 +261,4 @@ this folder (the most important two are first):
 
 
 Note that every package also has a local cache managed by the Cabal
-build system, e.g., in ``$distdir/cache``.
-
-There is another useful file in ``dist-newstyle/cache``,
```

Comment:
unintentionally rmeoved `plan.json` paragraph

Reply:
Actually It was intentional, as that file is mentioned in more structured paragraph just above this removed one.

Reply:
As you commented, if there are explanation above.